### PR TITLE
[WIP] First draft of backwards compatibility promise

### PR DESCRIPTION
**Pull request by MyDigitalLife**
So i made a small first draft of what could be the backwards compatibility promise based on [Semver](http://semver.org)  (#14) . Please feel free to provide feedback. Anything is welcome, things that should be added, things that should be specified in specific ways, things that need to be improved, etc.

### CODE REVIEW

**Comment by xmudrii** [comment]
Closes #14

File: docs/SEMVER.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Backwards compatibility promise
+
+This document is a work in progress. 
+The backwards compatibility promise will start as of version 1.0.0.
+Before this version new release can break backwards compatibility.
+
+## Semver
+
+Kubicorn follows [Semantic Versioning](http://semver.org/) when creating new releases.
```
Comment (prateekgogia) [inline docs/SEMVER.md]: Semantic Versioning 2.0.0

File: docs/SEMVER.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Every new release will be accompanied by a list of all bugs fixed and features added in that release. 
+Every bug and feature will have a link to the github issue associated with the change. 
+
+## Updating major releases
+
+When kubicorn decides to release a new major version we will include documentation that explains all backwards compatibility breaks.
+The documentation wil also provider recommended ways to upgrade your environment to the new version of Kubicorn. 
+Major updates don't have backwards compatibility by default.
+A major release can release without breaking backwards compatibility.
```
Comment (xmudrii) [inline docs/SEMVER.md]: Would it be better, from the grammar standpoint, to say `A major release can be released without breaking backwards compatibility`?

**Comment by kris-nova** [comment]
LGTM

**Pull request merged**